src/sentry/models/actor.py

```diff
@@ -170,7 +170,7 @@ def get_actor_tuple(self) -> ActorTuple:
     def get_actor_identifier(self):
```

Comment:
why do you need to keep this function around?

Reply:
This one is still being used. This is on the `Actor` model, I've changed the one on the `ActorTupl`